bento_beacon/config_files/config.py

```diff
@@ -18,12 +18,16 @@ class Config:
         "info": "record"
     }
 
+    BENTO_DOMAIN = os.environ.get("BENTOV2_DOMAIN")
     BEACON_BASE_URL = os.environ.get("BEACON_BASE_URL")
+    BENTO_PUBLIC_URL = "https://" + BENTO_DOMAIN
 
     # reverse domain id
-    BEACON_ID = ".".join(reversed(os.environ.get("BENTOV2_DOMAIN").split("."))) + ".beacon"
+    BEACON_ID = ".".join(reversed(BENTO_DOMAIN.split("."))) + ".beacon"
 
-    BEACON_NAME = os.environ.get("BENTO_PUBLIC_CLIENT_NAME", "BENTO") + " Beacon"
+    BEACON_NAME = os.environ.get("BENTO_PUBLIC_CLIENT_NAME", "Bento") + " Beacon"
+    BEACON_UI_ENABLED = os.environ.get("BENTO_BEACON_UI_ENABLED")
```

Comment:
maybe some logic to cast this to a boolean in the same way other services do it (check if `.lower()` string value in a set of predefined truth-y values) so we can set it to false if needed with predictable/consistent behaviour w/ other bento services